src/app.js

```diff
@@ -11,6 +11,12 @@ const config = require("./config");
 const app = express();
 
 const { origin, AppID, APP_SECRET, redirect_url, cookieAge } = config;
+if([origin, AppID, APP_SECRET, redirect_url, cookieAge].includes("")){
+  console.log("[WARNING] config.js has at least one empty property.")
+if ([origin, AppID, APP_SECRET, redirect_url, cookieAge].includes("")) {
+  console.warn("[WARNING] config.js has at least one empty property.");
+}
+}
```

Comment:
Code duplicated.

Reply:
I believe it was caused by just committing the added lines while forgetting to commit the removed ones as well. 

@Steven-Chan 
lines removed